var/spack/repos/builtin/packages/opa-psm2/package.py

```diff
@@ -29,10 +30,11 @@ class OpaPsm2(MakefilePackage):
     # patch to prevent opa-psm2 from adding an additional "usr/"
     #   subdirectory within the installation prefix, which breaks paths for
     #   dependent packages like libfabric
-    patch('opa-psm2-install-prefix.patch', when='@11.2.68:')
+    patch('opa-psm2-install-prefix.patch', when='@11.2.68:11.2.77')
+    patch('opa-psm2-install-prefix-11.2.185.patch', when='@11.2.185:')
```

Comment:
It is my understanding that these two patches do the same thing for different versions of the package. If that's the case, would it be feasible / easy to have a patch function filtering files for `/usr/lib` etc. entries instead of storing separate patches per package versions?

Reply:
Oh, interesting, I didn't know you could do that.  It turns out that the "edit" step already had a Makefile filter that didn't quite cover this case correctly.  I dropped the patch and updated that code block.  I just pushed the revision if you want to check.

Reply:
Thanks!